src/linear_algebra/affine_space/barycentric_coords.lean

```diff
@@ -4,9 +4,10 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Oliver Nash
 -/
 import linear_algebra.affine_space.independent
+import linear_algebra.determinant
 
 /-!
-# Barycentric coordinates
+# Affine bases and barycentric coordinates
```

Comment:
Should this file be renamed to `basis`?

Reply:
(perhaps in a future PR, as that might break the diff of this one)

Reply:
Yes, agreed on both points. If this is merged and does not rename the file, I will follow up proposing such a change.